runtime/nls/j9vm/j9vm_fr.nls

```diff
@@ -279,7 +279,7 @@ J9NLS_VM_CHECK_JAVA_HOME.user_response=Check that JAVA_HOME environment variable
 # This message will be followed by the exception's class name and
 # (if non-null) the exception's detail message
 # e.g. Exception in thread "main" java/lang/StackOverflowError: An overflow occurred
-J9NLS_VM_STACK_TRACE_EXCEPTION_IN=Exception dans l'unit\u00e9 d'ex\u00e9cution "%.*s"\u0020
+J9NLS_VM_STACK_TRACE_EXCEPTION_IN=Exception dans l'unit\u00e9 d'ex\u00e9cution "%.*s" 
```

Comment:
I don't think we want this sort of change.

Reply:
Why is that? It's what the automated translation process does, I don't want to have to change them back every time. The downside would be that it fails the line ending check.

Reply:
The check does indeed fail, but all the failures are for J9NLS_VM_STACK_TRACE_EXCEPTION_IN

Reply:
We should remove that part of the automation, or maybe better, arrange that messages don't need the trailing space.

Reply:
I have no control over the actual automation, although I could do my own post processing steps. 
Created https://github.com/eclipse-openj9/openj9/pull/18438